### PR TITLE
fix: localisation

### PR DESCRIPTION
**Pull request by okisdev**
This PR fixes the following issues:

- Localized Chinese README Translation
- Fixed some typos in the README

### CODE REVIEW

File: README_ZH.md [hunk 1]
```diff
@@ -150,35 +151,43 @@ print(interpreter.system_message)
 
 ### 更改模型
 
-ⓘ **在本地运行时遇到问题？** 阅读我们的新 [GPU设置指南](/docs/GPU.md) 和 [Windows设置指南](/docs/WINDOWS.md)。
-
-您可以从命令行在本地模式下运行 `interpreter` 以使用 `Code Llama`：
+在 `gpt-3.5-turbo` 下，使用更快的模式：
```
Comment (ifsheldon) [inline README_ZH.md:154]: "更快的模式" translates back as "faster mode", which is not faithful. "快速模式" is better

File: README_ZH.md [hunk 2]
```diff
@@ -215,44 +224,40 @@ INTERPRETER_CLI_DEBUG=False
...
-**⚠️ 在执行代码之前，Open Interpreter 会要求用户确认。**
+**⚠️ 所以在执行任何代码之前，Open Interpreter 都会要求用户确认是否运行。**
 
 您可以运行 `interpreter -y` 或设置 `interpreter.auto_run = True` 来绕过此确认，此时：
 
-- 在请求修改文件或系统设置的命令时要小心。
-- 观察 Open Interpreter 就像观察自动驾驶汽车，并准备通过关闭终端来结束进程。
-- 考虑在 Google Colab 或 Replit 这样的受限环境中运行 Open Interpreter。这些环境更加孤立，减少了执行任意代码的风险。
+-   在运行请求修改本地文件或系统设置的命令时要谨慎。
+-   请做好随时关闭终端以结束 Open Interpreter 进程的准备。
```
Comment (ifsheldon) [inline README_ZH.md:238]: The meaning is clear but the translation is not faithful. Back-translate as "be prepared to end the process by closing your terminal". Prepending "就像开自动驾驶汽车一样监看 Open Interpreter，" should be better.

**Comment by KillianLucas** [comment]
Great PR @okisdev. Excellent edits to the English docs, and thank you for reviewing/rewriting the Chinese README! Huge step towards humanity-wide access to the benefits of artificial intelligence.

**Comment by okisdev** [comment]
@KillianLucas Thanks for the recognition. Good to go now.

**Pull request merged**